src/seedu/addressbook/AddressBook.java

```diff
@@ -60,103 +62,103 @@
-    private static final String MESSAGE_WELCOME = "Welcome to your Address Book!";
-    private static final String MESSAGE_USING_DEFAULT_FILE = "Using default storage file : " + DEFAULT_STORAGE_FILEPATH;
-
-    // These are the prefix strings to define the data type of a command parameter
+    private static final String MESSAGE_INVALID_STORAGE_FILE_CONTENT   = "Storage file has invalid content";
+    private static final String MESSAGE_PERSON_NOT_IN_ADDRESSBOOK      = "Person could not be found in address book";
+    private static final String MESSAGE_ERROR_CREATING_STORAGE_FILE    = "Error: unable to create file: %1$s";
+    private static final String MESSAGE_ERROR_MISSING_STORAGE_FILE     = "Storage file missing: %1$s";
+    private static final String MESSAGE_ERROR_READING_FROM_FILE        = "Unexpected error: unable to read from file: %1$s";
+    private static final String MESSAGE_ERROR_WRITING_TO_FILE          = "Unexpected error: unable to write to file: %1$s";
+    private static final String MESSAGE_PERSONS_FOUND_OVERVIEW         = "%1$d persons found!";
+    private static final String MESSAGE_STORAGE_FILE_CREATED           = "Created new empty storage file: %1$s";
+    private static final String MESSAGE_WELCOME                        = "Welcome to your Address Book!";
+    private static final String MESSAGE_USING_DEFAULT_FILE             = "Using default storage file : "
+            + DEFAULT_STORAGE_FILEPATH;
+    
+    // These are the prefix strings to define the data type of a command
+    // parameter
     private static final String PERSON_DATA_PREFIX_PHONE = "p/";
     private static final String PERSON_DATA_PREFIX_EMAIL = "e/";
-
-    private static final String PERSON_STRING_REPRESENTATION = "%1$s " // name
-                                                            + PERSON_DATA_PREFIX_PHONE + "%2$s " // phone
-                                                            + PERSON_DATA_PREFIX_EMAIL + "%3$s"; // email
-    private static final String COMMAND_ADD_WORD = "add";
-    private static final String COMMAND_ADD_DESC = "Adds a person to the address book.";
-    private static final String COMMAND_ADD_PARAMETERS = "NAME "
-                                                      + PERSON_DATA_PREFIX_PHONE + "PHONE_NUMBER "
-                                                      + PERSON_DATA_PREFIX_EMAIL + "EMAIL";
-    private static final String COMMAND_ADD_EXAMPLE = COMMAND_ADD_WORD + " John Doe p/98765432 e/johnd@gmail.com";
-
-    private static final String COMMAND_FIND_WORD = "find";
-    private static final String COMMAND_FIND_DESC = "Finds all persons whose names contain any of the specified "
-                                        + "keywords (case-sensitive) and displays them as a list with index numbers.";
+    
+    private static final String PERSON_STRING_REPRESENTATION = "%1$s "                                             // name
```

Comment:
I guess you were trying to make the code smelly here. Please commit separately under the correct pull request next time